Framework/jmaqs-selenium/src/main/java/com/magenic/jmaqs/selenium/SeleniumConfig.java

```diff
@@ -141,39 +147,20 @@ public static String getBrowserName() {
   }
 
   /**
-   * Get the hub URL.
-   *
-   * @return the hub URL
-   */
-  public static String getHubUrl() {
-    return Config.getValueForSection(SELENIUM_SECTION, "HubUrl");
-  }
-
-  /**
-   * Get the web site base url.
-   *
-   * @return The web site base url
-   */
-  public static String getWebSiteBase() {
-    return Config.getValueForSection(SELENIUM_SECTION, "WebSiteBase");
-  }
-
-  /**
-   * Get the SavePageSourceOnFail flag from config.
-   *
-   * @return True if the flag is set to "Yes"
-   */
-  public static boolean getSavePagesourceOnFail() {
-    return Config.getValueForSection(SELENIUM_SECTION, "SavePagesourceOnFail").equalsIgnoreCase("Yes");
-  }
-
-  /**
-   * Get the SoftAssertScreenshot flag from config.
+   * Get the initialize Selenium timeout.
    *
-   * @return True if the flag is set to "Yes"
+   * @return The initialize timeout
+   * @deprecated Deprecated because specifying command timeout does not exist when creating web drivers in java.
    */
-  public static boolean getSoftAssertScreenshot() {
-    return Config.getValueForSection(SELENIUM_SECTION, "SoftAssertScreenshot").equalsIgnoreCase("Yes");
+  @Deprecated
+  public static int getCommandTimeout() {
```

Comment:
I think this can be removed permanently. It is not being used and the will not be used in the WebDriverFactory.